packages/renderer/MdToHtml/rules/link_close.ts

```diff
@@ -4,18 +4,25 @@
 import { RuleOptions } from '../../MdToHtml';
 import renderMedia, { Options as RenderMediaOptions } from '../renderMedia';
 
+
+export interface LinkIndexesType {
```

Comment:
LinkIndexes. Please look at the existing code, we don't add a "Type" suffix to types.